src/Symfony/Bundle/FrameworkBundle/Resources/config/serializer.php

```diff
@@ -129,8 +129,7 @@
                 service('property_info')->ignoreOnInvalid(),
                 service('serializer.mapping.class_discriminator_resolver')->ignoreOnInvalid(),
                 null,
-                null,
-                service('property_info')->ignoreOnInvalid(),
+                '$propertyInfoExtractor' => service('property_info')->ignoreOnInvalid(),
```

Comment:
Just realized, maybe using an abstract arg with a message mentioning the SerializerPass would've been a better option?